koku/masu/processor/ocp/ocp_cost_model_cost_updater.py

```diff
@@ -54,7 +54,9 @@ def __init__(self, schema, provider):
             )
             self._distribution_info = cost_model_accessor.distribution_info
 
-    def _build_node_tag_cost_case_statements(self, rate_dict, start_date, default_rate_dict={}, unallocated=False):
+    def _build_node_tag_cost_case_statements(
+        self, rate_dict, start_date, default_rate_dict={}, unallocated=False, node_core_type=False
```

Comment:
Why is the default value for `node_core_type` a boolean when the conditional is checking a string? The type of the default value should match the type which the function is expecting.

Reply:
This was originally a boolean and was used as such but changed while I was testing something else. Certainly should be changed back to match though, thanks :+1: